app/models/school.rb

```diff
@@ -154,11 +154,6 @@ def slug_candidates
     ]
   end
 
-  # This is temporary so that nothing breaks until the slugs are generated
-  def to_param
-    id
-  end
-
```

Comment:
So we can't merge this PR until we do a prod release and run the script for slugifying urls for all schools, right?

Reply:
it'll be merged once i've done the _staging_ release and script, develop will become undeployable for a bit, then i'll deploy the first release to production, run the script, then deploy the second release (and develop will become deployable again)